wordpress-mobile/WordPress-iOS
# Move app settings to a new section.

Fixes #5108 

To test:

 * Go to the Me tab and check for the presence of the new App Settings.
 * This should be tested on the following scenarios:
   * Logged to WP.com
   * Logged to self-hosted site.
   * Logged to both

Needs review: @koke can you please review?

Please double check if the code that I added to Account Setting to show a loading information makes sense and if I'm using Imutable correctly on App Settings
